### DISCUSSION TITLE
Kenapa url $pager yang muncul url $pager parent nya?

### DISCUSSION THREAD

**Discussion by yuliakbard** [mecha-cms/mecha#221] [Q&A]
![Screenshot 2022-06-07 125906](https://user-images.githubusercontent.com/96925342/172306689-06a20422-c8b2-48d3-a0bd-bf867ccac632.jpg)
ini url pada $pager kenapa yang muncul seperti ini kang?

Contoh:

pada `https://lokersumatera.my.id/loker/pekanbaru` harusnya yang muncul url `$pager` = `https://lokersumatera.my.id/loker/pekanbaru/2`

tetapi yang muncul nilai $pager nya:
`https://lokersumatera.my.id/loker/2`

Maaf kang, sulit saya jelasinnya gimana. coba diliat pada gambarnya juga

**Comment by taufik-nurrohman** [comment]
Saya pengin tau nilai atribut `class` pada elemen `<html>` di laman tersebut.

**Comment by yuliakbard** [comment]
![Screenshot_20220607-165247_Chrome](https://user-images.githubusercontent.com/96925342/172351747-45e3d6c5-132f-4f4c-8971-3a9c219b0b9f.jpg)

ini kang

**Reply by taufik-nurrohman** [reply]
Hmmm belum pernah mengalami. Coba pakai tema lain efeknya bener apa sama kaya gitu. Untuk testing aja.

**Reply by yuliakbard** [reply]
ada alternatif lain gak kang? di custom aja gitu kang di ambil dari url yang ada. saya mau coba pake f3h infinite scroll

**Reply by taufik-nurrohman** [reply]
Infinite scroll juga kan perlu navigasi _next_. Tapi kalau mau ngambil dari URL bisa juga si:

~~~ js
let {href} = window.location;
href = href.split(/[?&#]/)[0];
let offset = 1;
    m = href.match(/\/(\d+)$/), next;
if (m && m[1]) {
    offset = +m[1];
    next = href.replace(/\/\d+$/, '/' + (offset + 1));
} else {
    next = href + '/' + (offset + 1);
}

console.log(next);
~~~

**Reply by yuliakbard** [reply]
Kang, sepertinya ini semacam error di mecha 2.6.4. ane coba dilocal host dengan struktur halaman yang serupa.

`localhost/test/lowongan-kerja/pekanbaru/` yang seharusnya url next pager nya tuh `localhost/test/lowongan-kerja/pekanbaru/2` juga jadi `localhost/test/lowongan-kerja/2`

![Screenshot 2022-06-10 141046](https://user-images.githubusercontent.com/96925342/173010482-762ed0ba-fa3b-4f54-98b0-0e87ea07acfd.jpg)

dan ini di mecha versi 3.0.0 yang ada di github, yang pernah ane download di 25 Maret 2022, disini url next pager nya normal kang `localhost/test/lowongan-kerja/pekanbaru/2`

![Screenshot 2022-06-10 141935](https://user-images.githubusercontent.com/96925342/173012047-d4ed2ce8-7cf9-431f-81ef-83704e72d03d.jpg)
